boards/renesas/ek_ra8d1/ek_ra8d1.dts

```diff
@@ -42,6 +42,7 @@
 
 	aliases {
 		led0 = &led1;
+		sdhc0 = &sdhc1;
```

Comment:
Need to add `-sdhc` to the boards' test YAML so the SDHC tests will actually run

Reply:
Added `-sdhc` into the boards' yaml.